### DISCUSSION TITLE
Hello dear ones. I used this piece of code to practice js and it gives me this error. I wanted to know where the problem is with this code

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69900] [Programming Help] [answered]
### Body

Erorr
  .Uncaught TypeError: EVT.preventdefault is not a function     onClick file:///E:/programing/javascript-webprog/tamrin/tamrin.js:718     EventListener.handleEvent* file:///E:/programing/javascript-webprog/tamrin/tamrin.js:720

code html 
<input type="submit" id="input">

code js
const input = document.getElementById("input");
const onClick = function (EVT) {
  EVT.preventdefault();
  console.log(EVT.target);
};
input.addEventListener("click", onClick);

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by thevindu-w** [comment, marked as answer]
It should be `EVT.preventDefault();` Note that the 'D' is in uppercase.
